tasks/config/webpack.config.js

```diff
@@ -0,0 +1,108 @@
+var webpack = require('webpack');
+var path = require('path');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  // Where to start
+  entry: {
+    main: path.resolve(__dirname, '../../src/js/index.js')
```

Comment:
Let's change main to something else like "ShipComponentsDropzone". Otherwise in dist we'll have a bunch of files called main.js which isn't terribly descriptive.